packages/web-app-files/src/helpers/resource/resource.ts

```diff
@@ -57,3 +58,24 @@ export const extractStorageId = (id?: string): string => {
   }
   return id.indexOf('!') >= 0 ? id.split('!')[0] : ''
 }
+
+export const extractNameWithoutExtension = (resource?: Resource): string => {
+  const extension = resource.extension || ''
+  const name = resource.name || ''
+  if (!extension.length || !name.length) return name
```

Comment:
```suggestion
  if (!extension.length) return name
```
should be sufficient, right?